include/libtorrent/string_view.hpp

```diff
@@ -41,11 +41,13 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/utility/string_ref.hpp>
 namespace libtorrent {
 using string_view = boost::string_ref;
+using wstring_view = boost::wstring_ref;
```

Comment:
I think this should exists only in `#if TORRENT_USE_WSTRING || defined TORRENT_WINDOWS`